src/ray/protobuf/gcs.proto

```diff
@@ -94,7 +94,7 @@ message TaskTableData {
   Task task = 1;
 }
 
-message ActorTableData {
+message ActorStates {
```

Comment:
`ActorStates` seems to be the plural of `ActorState`, which is confusing.

Reply:
`ActorStates` including several states of actor, including state, timestamp, address, etc. It kinda confusing and better naming is welcome!